src/main/java/org/hiero/sketch/table/PartialMembershipDense.java

```diff
@@ -65,6 +73,81 @@ public int getSize() {
     }
 
     @Override
+    public int getSize(boolean exact) {
```

Comment:
I actually like this interface, it does not clutter the API. I thing of getSize() as just setting the default to true (that's how I would write it in C++ where there are defaults). I don't see a reason to provide confidence intervals. We could have getAppSize(int k) where k is the number of samples used to drive the approximation, but really I don't see a real reason for this right now.